fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -2182,40 +2243,24 @@ ACTOR Future<Void> BgDDLoadRebalance(DDQueue* self, int teamCollectionIndex, Dat
 		state GetTeamRequest destReq;
 		state TraceEvent traceEvent(eventName, self->distributorId);
 		traceEvent.suppressFor(5.0)
-		    .detail("PollingInterval", SERVER_KNOBS->BG_REBALANCE_POLLING_INTERVAL)
+		    .detail("PollingInterval", rebalancePollingInterval)
 		    .detail("Rebalance", readRebalance ? "Read" : "Disk");
 
+		// NOTE: the DD throttling relies on DDQueue, so here just trigger the balancer periodically
+		wait(delay(rebalancePollingInterval, TaskPriority::DataDistributionLaunch));
```

Comment:
This change would only make a difference the very first time that this actor runs on behalf of each MC/VF, yes?

Reply:
Yes. I think in the past we first read the skip value from the database and then delay. Now it becomes the first delay and then read from the database. But I think it isn't a huge difference given the rebalance is not a time-sensitive thing.